Dockerfile

```diff
@@ -23,21 +23,28 @@ RUN apt-get -yq install --no-install-recommends \
         tcpdump traceroute iproute2 dnsutils whois mtr iftop iputils-ping wget nmap netcat-traditional \
         procps \
         htop \
-    # Clean up
-    && apt-get autoremove -y \
-    && apt-get clean -y \
-    && rm -rf /var/lib/apt/lists/* 
+        screen tmux \
```

Comment:
These are all very good additions and can make a general improved experience for all users of this toolbox.